cmd/siac/main.go

```diff
@@ -118,5 +233,113 @@ func main() {
 		} else {
 			fmt.Printf("%s SC\n", balance.Siacoins)
 		}
+
+	case walletAddressCmd:
+		if len(args) != 0 {
+			cmd.Usage()
+			return
+		}
+		address, err := getClient().WalletAddress()
+		check("couldn't get address:", err)
+		fmt.Println(address.Address)
+
+	case walletAddressesCmd:
+		if len(args) != 0 {
+			cmd.Usage()
+			return
+		}
+		addresses, err := getClient().WalletAddresses()
+		check("couldn't get address:", err)
+		for _, address := range addresses.Addresses {
+			fmt.Println(address)
+		}
+
+	case walletTransactionsCmd:
+		if len(args) != 0 {
+			cmd.Usage()
+			return
+		}
+		transactions, err := getClient().WalletTransactions()
+		check("couldn't get transactions:", err)
+
+		fmt.Fprintf(w, "%v\t%v\t%v\t%v\n", "Index", "ID", "Inflow", "Outflow")
+		for _, txn := range transactions.Transactions {
+			fmt.Fprintf(w, "%v\t%v\t%v\t%v\n", txn.Index, txn.ID, txn.Inflow, txn.Outflow)
+		}
+
+	case walletSignCmd:
+		if len(args) != 1 {
+			cmd.Usage()
+			return
+		}
+
+		var txn types.Transaction
+		check("failed to decode JSON:", json.Unmarshal([]byte([]byte(args[0])), &txn))
```

Comment:
redundant `[]byte` here

We'll also want to support reading/writing these transactions from disk